Makefile

```diff
@@ -29,21 +29,22 @@ mocks:
 	@go generate ./...
 
 build: clean lint
-	@echo Build ${BINARY}
-	@go build ./...
-	@go build -o ./bin/${BINARY} ./cmd/monaco
+	GOOS=windows GOARCH=amd64 CGO_ENABLED=0 go build -tags netgo -ldflags '-w -extldflags "-static"' -o ./bin/${EXECUTABLE}-windows-amd64.exe ./cmd/monaco
+	GOOS=windows GOARCH=386 CGO_ENABLED=0 go build -tags netgo -ldflags '-w -extldflags "-static"' -o ./bin/${EXECUTABLE}-windows-386.exe ./cmd/monaco
```

Comment:
the release build will miss these as it's uploading assets from a `build` folder, not `bin`, pls adapt one of them